.github/release-drafter.yml

```diff
@@ -0,0 +1,42 @@
+name-template: "v$RESOLVED_VERSION 🌟"
+tag-template: "v$RESOLVED_VERSION"
+
+categories:
+  - title: "Features"
+    labels: ["feature"]
+  - title: "Bug Fixes"
+    labels: ["bug"]
+  - title: "Chores"
+    labels: ["chore"]
+  - title: "Documentation"
+    labels: ["documentation"]
+  - title: "Dependency Updates"
+    labels: ["dependencies"]
+
+change-template: "- $TITLE (#$NUMBER) by @$AUTHOR"
+change-title-escapes: '\<*_&'
+version-resolver:
+  minor:
+    labels:
+      - "feature"
+  patch:
+    labels:
+      - "bug"
+      - "documentation"
+      - "dependencies"
+      - "chore"
+
+exclude-labels:
+  - "skip-changelog"
```

Comment:
`exclude-label` means that PR with this label will not have effect on the release notes. Therefore `skip-changelog` is here enough. But ensure that `skip-changelog` is on our label list.